### DISCUSSION TITLE
Carla fork of Unreal Engine 4.26 no longer exists

### DISCUSSION THREAD

**Discussion by o-antill** [carla-simulator/carla#7774] [Q&A] [answered]
Hi there, 
I am currently trying to build Carla from source however when following the given guide on the official Carla documentation (https://carla.readthedocs.io/en/0.9.15/build_linux/) I am unable to find and or install the Unreal Engine 4.26 version of Carla (Saying the page no longer exists). 

I have linked my Github account and my Unreal engine account using the same email and have gained access to the Unreal engine repository via an email link. 

Currently I am on Ubuntu 20.04LTS if that is any help 
![Screenshot from 2024-06-11 15-09-30](https://github.com/carla-simulator/carla/assets/171234048/7910ea59-6f2f-4ddd-a151-f9541eb10072)

**Comment by catac0mb** [comment]
Try using SSH key instead of username and password authentication: 
1. https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
2. https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

Replace the documentation’s cloning command with the SSH key cloning command git clone --depth 1 -b carla git@github.com:CarlaUnreal/UnrealEngine.git ~/UnrealEngine_4.26 and proceed with the documentation instruction for the rest of the steps.

**Reply by o-antill** [reply]
I believe it may be because i dont have the correct access rights to the correct repository no ? When i was added to the Unreal Engine github repository i was added to a version of it called Mirror A rather than the original unreal engine repository.
![Screenshot from 2024-06-13 13-38-12](https://github.com/carla-simulator/carla/assets/171234048/220c6e49-6cc3-40e7-8ea2-3939708745ac)

**Reply by o-antill** [reply, marked as answer]
Im guessing something to do with this ...
![Screenshot from 2024-06-13 14-04-18](https://github.com/carla-simulator/carla/assets/171234048/43d675a8-20bd-46ef-a881-01ce1ea52a88)
